enaml/qt/qt_mpl_canvas.py

```diff
@@ -89,11 +89,17 @@ def _refresh_mpl_widget(self):
         # visible, or QVBoxLayout will ignore it for size hinting.
         figure = self.declaration.figure
         if figure:
```

Comment:
What happens if there is no `figure` (i.e. it was set to None in Enaml)? Does the old stale figure continue to be displayed?